packages/fxa-auth-server/lib/server.js

```diff
@@ -401,6 +401,19 @@ async function create(log, error, config, routes, db, statsd, glean) {
     hawkFxAToken.strategy(makeCredentialFn(db.passwordChangeToken.bind(db)))
   );
 
+  server.auth.scheme(
```

Comment:
Cool, I like this. I think a comment above might be helpful to explain how these are used.